cli/tests/pcluster/config/test_section_aws.py

```diff
@@ -14,6 +14,23 @@
 from pcluster.config.mappings import AWS
 
 
+@pytest.fixture()
+def default_region_mock():
+    """
+    Mock default of aws_region_name in AWS dict.
+
+    The default aws_region_name depends on default region in aws config file, therefore varies.
+    Mock it to make sure tests have consistent behavior when running on different machines
+    """
+    original_region = AWS["params"]["aws_region_name"]["default"]
+
+    def _default_region_mock(region=None):
+        AWS["params"]["aws_region_name"]["default"] = region
+
+    yield _default_region_mock
+    AWS["params"]["aws_region_name"]["default"] = original_region
+
+
```

Comment:
I had to duplicate this fixture in ``test_pcluster_config_convert`` because in Python2.7, I can not read ``AWS`` from ``mappings.py`` in ``conftest.py``

Reply:
Let's add this comment in the code.

Not clear to me why it depends by Python version?

Reply:
If we write ``from pcluster.config.mappings import AWS`` in ``conftest.py``, 
Python 2.7 will complain ``ImportError: No module named mappings``